### ISSUE TITLE
UX analytics: How are users navigating the portal?

### ISSUE THREAD

**Issue by fmccaf1** [department-of-veterans-affairs/va.gov-team#90192]
## Description

High prio (from eval plan ratings):
How are users navigating the portal?

### User story

As a Cartography team member, I want to understand how users are navigating the portal, keeping in mind breadcrumbs, links on the MHV landing page, secondary nav, and any other ways users can move between tools.

### Notes
- Use GA4
- The point is to be able to compare use of breadcrumbs vs. secondary nav vs. back button vs. landing page
- Anne CK might be a contact for help

Possible tasks:
- [x] Attend analytics office hours and/or watch past trainings to gain understanding 
- [ ] Validate what you create w Jamie Stutt and / or Wes Rowe

## Acceptance criteria
- [x] Report(s) is set up to track navigation methods thru portal

**Comment by wesrowe** [comment]
@fmccaf1 @katbrinkley @jonathanjnelson @sterkenburgsara – I think we could refine this ticket in more detail to focus it on the specific paths we want to measure. And also, how we want to present it. For example, we already figured out secondary nav clicks, but what format would we use to compare those numbers to, say, breadcrumb clicks? Maybe it's starting with just a mural, a really simple flow diagram with numbers next to arrows? The word "Report" in the existing AC makes me think this deliverable is more formal than the first-draft swipe that I would think is the intent.

Mainly I want to spare us from a ticket where the person who picks it up doesn't have a really clear path to bang it out and move on to other pastures.

Happy to discuss more via slack. :-)

**Comment by wesrowe** [comment]
May be blocked for s38 due to missing GA events for header. (Maybe all of header is missing??)

**Comment by fmccaf1** [comment]
There are multiple ways to get to answers to these questions. I believe that Robyn looks at SN use overall to (at least partially) understand what percent of users are clicking on the secondary nav during their session. We could continue to use a metric like this one and we could also get more specific I think. 

Here are some findings that would be easy to continue to collect as needed:
- On the most basic level, users are not sticking to the navigational strategies they used in the national portal.

**Comment by fmccaf1** [comment]
### /My-health/appointments and /my-health/appointments/ are the only top 1 or 2 level tool pages on which users tend to prefer breadcrumbs as their method to get to MHV home (on VA.gov). 
On other tool pages including secure-message/inbox/, records, medications/refill, medications/, appointments/schedule/type-of-care, medications/about, and secure-messages, users use the secondary nav link more than they use the breadcrumbs link to get to the landing page

**Comment by sterkenburgsara** [comment]
WOW! Very cool. Great job, @fmccaf1

**Comment by fmccaf1** [comment]
Also wrote these docs to help with GA4 onboarding:
- [Tips for isolating events on the landing page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/metrics-performance-monitoring/GA4-events-and-guidance.md) (or explained differently, the secret "codes" to tell GA4 the types of clicks we want to know about in any report)
- [Step-by-step report building for our eval questions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/landing-page/research/how-to-eval-plan.md)

Both of the above are still in process

**Comment by fmccaf1** [comment]
[Based on numbers I exported to this excel sheet](https://agile6-my.sharepoint.com/:x:/p/florence_mccafferty/EX2iJaezgztKtotjqwvrwPcBWmZjpLlU_bDxpz5kFqHI_A?e=meRdbE), a few findings on clicks on the landing page. Note: I removed all records-related clicks from the data to get a clearer comparison.

Dates: July 1-31, 2024

- The number of clicks on links in cards is much higher than clicks on Secondary Nav. When comparing totals of link clicks in cards (except records) and total clicks on secondary nav (except records), Secondary Nav makes up 16% of clicks. Appointments card receives the most clicks, at 27%, then Messages at 26%, Medications at 21%. Payments and Medical supplies cards received 6% and 4% respectively.
